toolbox/linux/which.rst

```diff
@@ -0,0 +1,111 @@
+Which Linux Should I Use?
+=========================
+
+Unlike Windows or OS X, there are several different "flavors" of Linux you can
+use. We call them "`Linux distributions
+<https://en.wikipedia.org/wiki/Linux_distribution>`_."
+
+Distributions?
+--------------
+
+An operating system is made up of many different pieces that come together to
+form a whole. There's the "kernel" (the core of the operating system), which is
+always Linux. But there are lots of other different parts, like the user
+interface, software for managing network connections, etc.
+
+A Linux distribution simply chooses all of the pieces required to form a usable
+operating system. The distribution glues everything together, making sure
+everything works properly.
+
+Linux distributions also provide ways for you to install software. On OS X or
+Windows, when you need software you probably search the web, download an
+installer, and run it. Linux distributions, on the other hand, make their own
+"installers" for most programs you might want to use. They usually have "package
+managers" that let you install these programs with a single command. This is
+typically easier, safer, and faster than installing (and updating) software on
+OS X or Windows.
+
+There are tons of choices of distribution, with names like Ubuntu, Debian, Mint,
+Fedora, Arch Linux, and Gentoo (to name a few). With all these choices, how do
+you know which one to use?
+
+**If it's your first time using Linux, use Ubuntu.** It's a fairly friendly
+place to start, and there's tons of help available for it. Check it out `here
+<http://www.ubuntu.com/>`_, or go straight to the `downloads page
+<http://www.ubuntu.com/download/desktop>`_ (you don't have to give them money,
+just scroll past their donation stuff).
+
+Some people don't like Ubuntu's default user interface (called Unity).
+Thankfully, Ubuntu comes with a few different interfaces. These are the two most
+popular. These are just fine to use for your first time using Linux as well. You
+might want to Google for some screenshots to see which one you might like.
+- `Ubuntu GNOME <https://ubuntugnome.org/>`_.
+- `Kubuntu <http://www.kubuntu.org/>`_ (that's Ubuntu + KDE)
+
+This ain't my first rodeo
+-------------------------
+
+If you've used Ubuntu quite a bit and you're familiar with the command line, you
+may want to try something new and exciting.  Here are a few others:
+
+Linux Mint
+~~~~~~~~~~
+
+`Linux Mint <https://www.linuxmint.com/>`_ is a distribution based on Ubuntu
+that tries to be *even easier* than regular Ubuntu. You probably won't learn a
+ton by switching to Mint from Ubuntu, but it's a new look and an easy to use
+distro.
+
+Arch Linux
+~~~~~~~~~~
+
+For many who have familiarized themselves with Linux and want to learn more,
+`Arch <https://archlinux.org>`_ is a great next step. This distribution always
+has the latest, most up to date software. You can use nearly any desktop you'd
```

Comment:
really I mean "desktop environment".  "desktop manager" is a dangerous term to use because it confuses the two distinct things "display manager" and "desktop environment"